presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/RemoveUnsupportedDynamicFilters.java

```diff
@@ -241,7 +241,8 @@ private Expression removeAllDynamicFilters(Expression expression)
             return combineConjuncts(metadata, extractResult.getStaticConjuncts());
         }
 
-        private Expression removeNestedDynamicFilters(Expression expression) {
+        private Expression removeNestedDynamicFilters(Expression expression)
+        {
```

Comment:
Ah, I see you undid the change in the first commit. Please squash this one in there.